.gitignore

```diff
@@ -20,3 +20,5 @@ __pycache__/
 .pio/
 
 test/*/test_*
+
+.clang-format
```

Comment:
.clang-format shouldn't be gitignored, it should be committed to the repo

Reply:
Yes sorry, I added some extra config while doing a quick test in my IDE and forgot to remove that line